routes/user.js

```diff
@@ -33,10 +33,10 @@ module.exports = function(app) {
           const loggedUser = await Model.login(req.body)
           if (loggedUser) {
             req.session.ckan_user = loggedUser
-            req.flash('info', 'Testing flash messages')
             res.redirect("/profile")
           } else {
-            res.redirect("/login-failure")
+            req.flash('info', 'Login failed.')
```

Comment:
That's fine for now. We might need to provide details of the failure in the future.